README.md

```diff
@@ -60,6 +60,7 @@ Refer [actions-setup](https://github.com/variant-inc/actions-setup) for document
        with:
          dockerfile_dir_path: '.'
          ecr_repository: naveen-demo-app/demo-repo
+         github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
With this change, we will probably need to make a post in cloudops-announcements that this is a potentially breaking change for some repos, and they will need to to include the github_token when we update the tag.

Reply:
Can this not be added as an environment variable instead of having each workflow pass it in as input?

Reply:
@salyateem13 I think Vibin requested it to be an input since he also asked for this in actions-cpp

Reply:
yes we discussed to have input.